### DISCUSSION TITLE
Post on Monday

### DISCUSSION THREAD

**Discussion by chenilim** [mattermost/community#56] [General Ideas]
This probably already exists, but I can't find it - sometimes (more often than I would like), I want to post something late Friday, or even on the weekend, but don't want it to show up until Monday morning. Ideally, there should be a dropdown on posts to queue them up for a certain time (e.g. 9am Monday).

**Comment by aedott** [comment]
@jespino worked on something like this for one of the hackathons last year! https://www.youtube.com/watch?v=VKVrgXzWO1M

**Reply by jespino** [reply]
Yes, the plugin is taking dust in my personal github, if anyone wants to undust it, I'm happy to transfer the ownership of the repo.

**Reply by darkLord19** [reply]
Can you share the link of the plugin and current status of it? I may undust it a bit once I get some time as I am quite interested in timing/scheduling related problems lately1

**Reply by jespino** [reply]
https://github.com/jespino/mattermost-plugin-messages-queue

This was a bit hackathon thing, but feel free to take it and evolve it. Let me know if you want me to transfer it to your user in GitHub

**Reply by darkLord19** [reply]
@jespino I guess that's not necessary. Will just fork your repo and see work on it.

**Comment by esethna** [comment]
I wonder if having a notifications schedule for DND would help this pain point in a different way? So there isn't as much anxiety about sending the message

**Reply by jespino** [reply]
Yes, that works, but sometimes is not only the DND status, it is also something like people offline receiving email notifications and answering to it.
